### DISCUSSION TITLE
The background grass tiles in the forest look too similar to the foreground's

### DISCUSSION THREAD

**Discussion by Dragonick-T-Hedgewing** [SuperTux/supertux#2303] [Ideas]
![image](https://user-images.githubusercontent.com/98646446/194119263-7e28a3e5-c57f-4965-a915-2052bf48078d.png)

The background tiles in some of the Forest stages have grass on them that looks too similar to that in the foreground. This sometimes throws me off; whilst I've gotten sort of used to it, I've missed some jumps or fallen into the water because of it.

They aren't identical, but they're similar enough to fool a quick or distracted player, and they sort of look wrong with how dark the rest of the background tiles are in comparison.

**Comment by Rusty-Box** [comment]
This will be changed with the new tileset overhaul #2136 
Background walls with grassy tops will be unisolid going forward.
